zio-http/src/main/scala/zio/http/codec/HttpCodec.scala

```diff
@@ -195,6 +195,14 @@ sealed trait HttpCodec[-AtomTypes, Value] {
   ): Z =
     encoderDecoder.encodeWith(value)(f)
 
+  def examples(examples: Iterable[Value]): HttpCodec[AtomTypes, Value] =
+    HttpCodec.WithExamples(self, Chunk.fromIterable(examples))
+
+  def examples(examples: Value*): HttpCodec[AtomTypes, Value] =
```

Comment:
To improve type inference:
```suggestion
  def examples(example1: Value, examples: *): HttpCodec[AtomTypes, Value] =
```